<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveProperty</class>
 <widget class="QDialog" name="CurveProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>curve configuration</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QListWidget" name="list_selected_item">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择需要观察的曲线：</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_unselected_item">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>181</width>
        <height>192</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>curve1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>curve2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>curve3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>curve4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>curve5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>curve6</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>241</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数据采集设置：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="end_range">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>120</y>
        <width>75</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>终止数据</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>84</y>
        <width>75</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>起始数据</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="start_range">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>84</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>坐标轴属性</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>241</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x 轴最小值：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_min">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_max">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>86</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x 轴最大值:</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y 轴最小值：</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="y_min">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="y_max">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>86</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y 轴最大值:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_axis_set">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>270</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>保存设置</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
